tests/core/test_exporter.py

```diff
@@ -73,9 +73,9 @@ async def test_fetch_events_within_time_range():
     conversation_ids = ["some-id", "another-id"]
 
     # prepare events from different senders and different timestamps
-    event_1 = random_user_uttered_event(3)
-    event_2 = random_user_uttered_event(2)
-    event_3 = random_user_uttered_event(1)
+    event_1 = random_user_uttered_event(1)
+    event_2 = random_user_uttered_event(3)
+    event_3 = random_user_uttered_event(2)
```

Comment:
The arguments here refer to the timestamp of events. The test here was to assume that,
`conversation1 = [event1, event2]` where `event2` happens before event1. However this is not a fair assumption since the event timestamps are not externally assigned but depend on the timestamp of machine -- as defined here https://github.com/RasaHQ/rasa/blob/3.6.x/rasa/shared/core/events.py#L284

Reply:
I'm trying to understand why the initial test setup assumed the order of events is `[event1, event2]` if event2 happens first? Was that to test that the timestamp ordering happens? If so, why do you need to alter the test if you've sorted the events by timestamp in the exporter code?

Reply:
@ancalita I'm not sure why did the test assumed that order of events. The changes we've made in the Tracker Store are only limited to SQL Tracker Store. However there's an in-memory tracker store being used here. My additional changes to sort events in exporter still apply. The test fails with the following diff,

```
>           assert event_timestamps == [e.timestamp for e in events[_id]]
E           assert [2, 3] == [3, 2]
E             At index 0 diff: 2 != 3
E             Full diff:
E             - [3, 2]
E             + [2, 3]
```

Which is, that it is trying to assert that the order of events returned is the same as given. I changed the given here since it is a false test case.